Source/Core/RectangleCollisionChecker.js

```diff
@@ -22,11 +26,14 @@ define([
         this.id = '';
     }
 
-    RectangleWithId.fromRectangleAndId = function(id, rectangle, result) {
-        result.minX = rectangle.west;
-        result.minY = rectangle.south;
-        result.maxX = rectangle.east;
-        result.maxY = rectangle.north;
+    var projectedExtentsScratch = new Rectangle();
+    RectangleWithId.fromRectangleAndId = function(id, rectangle, result, mapProjection) {
```

Comment:
Needs documentation here.

Reply:
Should be fine since this is internal to a private class.